components/ui/Loading.js

```diff
@@ -0,0 +1,46 @@
+import styled, { css, keyframes } from 'styled-components'
+
+const size = 16
+const sizePx = `${size}px`
+const beforeAfterOffset = size * 2
+const defaultBackgroundColor = '#8f8d76'
+
+const flashKeyframes = percentages => keyframes`
+  ${percentages} {
+    background-color: #fffcd8;
```

Comment:
This should probably grab from the theme: `background-color: ${props => props.theme.colors.vibrantPixel};`